src/Microsoft.PowerShell.Commands.Utility/commands/utility/select-object.cs

```diff
@@ -339,8 +339,8 @@ private void ProcessExpressionParameter()
             if (ExcludeProperty != null)
```

Comment:
@iSazonov Good point. I just searched how `_exclusionFilter` is used and it turns out we don't need to initialize it with [line#341](https://github.com/PowerShell/PowerShell/pull/2602/files/e695327c85646d9eb76c70b6dd68c83a73126cde#diff-e3887c1e2440a9de83bea8869a9048d1R341) when the length is 0. I will make the change.